apis/management.cattle.io/v3/k8s_defaults.go

```diff
@@ -55,13 +55,23 @@ var (
 			},
 			Kubelet: map[string]string{
 				"tls-cipher-suites": "TLS_ECDHE_ECDSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_ECDSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_ECDSA_WITH_CHACHA20_POLY1305,TLS_ECDHE_RSA_WITH_AES_128_GCM_SHA256,TLS_ECDHE_RSA_WITH_AES_256_GCM_SHA384,TLS_ECDHE_RSA_WITH_CHACHA20_POLY1305",
+				"cadvisor-port":     "0",
 			},
 		},
 		"v1.9": {
 			KubeAPI: map[string]string{
 				"endpoint-reconciler-type": "lease",
 				"admission-control":        "ServiceAccount,NamespaceLifecycle,LimitRanger,PersistentVolumeLabel,DefaultStorageClass,ResourceQuota,DefaultTolerationSeconds",
 			},
+			Kubelet: map[string]string{
+				"cadvisor-port": "0",
+			},
+		},
+		"v1.8": {
```

Comment:
we no longer support k8s 1.8, so that can be dropped

Reply:
We did pull out the support; it is driven by https://github.com/rancher/types/blob/master/apis/management.cattle.io/v3/k8s_defaults.go#L18 for open source users, and SLA for the customers.

Reply:
Removed 1.8 block